usbsdmux/usb2642i2c.py

```diff
@@ -72,15 +74,33 @@ class Usb2642I2C(object):
   Make sure you have rw-rights :)
   """
 
-  def __init__(self, sg):
+  _USB_VALIDATION_FIELDS = (
+    'idVendor', 'idProduct', 'manufacturer', 'product'
+  )
+  """
+  Fields in the USB device descriptor that are checked
+  to identify valid USB-SD-Muxes
+  """
+
+  _USB_VALIDATION_VALUES = (
+    ('0424', '4041', 'Pengutronix', 'usb-sd-mux_rev2.0'),
+    ('0424', '4041', 'Linux Automation GmbH', 'usb-sd-mux_rev4.0')
+  )
```

Comment:
The idVendor/idProduct are not sufficient to validate that we are operating on a sd-mux here?

Reply:
Actually not, as the `idVendor` and `idProduct` are shared by all card readers based on the `USB2642` chip, most of which are not USB-SD-Muxes. Using the `usbsdmux` command on one of these would result in messages being sent by the `USB2642` on its `I2C` bus, which would most likely be okay, but we can not guarantee that.